test/test_stream.rb

```diff
@@ -89,6 +89,44 @@ def entity(content)
     end
   end
 
+  class EntityExpansionLimitTest < Test::Unit::TestCase
+    def setup
+      @default_entity_expansion_limit = REXML::Security.entity_expansion_limit
+      @default_entity_expansion_text_limit = REXML::Security.entity_expansion_text_limit
+    end
+
+    def teardown
+      REXML::Security.entity_expansion_limit = @default_entity_expansion_limit
+      REXML::Security.entity_expansion_text_limit = @default_entity_expansion_text_limit
+    end
+
+    def test_with_only_default_entities
+      member_value = "&lt;p&gt;#{'A' * @default_entity_expansion_text_limit}&lt;/p&gt;"
+      source = StringIO.new(<<-XML)
```

Comment:
```suggestion
      source = <<-XML
```
I think StringIO.new is unnecessary.

Reply:
Fixed in [b7e9378](https://github.com/ruby/rexml/pull/199/commits/b7e937898847d4ab938ade808f57e47a1d08975e)